src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js

```diff
@@ -0,0 +1,52 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeSupportResourcesDetailPage on NodeSupportResourcesDetailPage {
+  ${entityElementsFromPages}
+  entityBundle
+  changed
+  title
+
+  fieldContentBlock {
+    entity {
+      entityType
+      entityBundle
+      ... wysiwyg
+      ... collapsiblePanel
+      ... process
+      ... qaSection
+      ... qa
+      ... listOfLinkTeasers
+      ... reactWidget
+      ... spanishSummary
+      ... alertParagraph
+      ... table
```

Comment:
Good thinking to do the table improvements in a future PR instead of this one (enough going on)

Reply:
I think we actually got the go-ahead to make the table update at a site-wide level - https://dsva.slack.com/archives/C0173CAJ7A6/p1601398279071400?thread_ts=1601326427.054300&cid=C0173CAJ7A6 - so good stuff!

Reply:
sweet! 🔥